.github/workflows/causal_pipeline_uncensored.yml

```
@@ -39,37 +39,41 @@ jobs:
-          run_step main.prepare_training_data prepare_and_pretrain/prepare_pretrain.yaml
-          run_step main.pretrain prepare_and_pretrain/pretrain.yaml
-          run_step main.create_outcomes outcomes.yaml
-          run_step main_causal.select_cohort_full select_cohort_full/extract.yaml
-
-      - name: Prepare Finetune Data
+      - name: Outcomes & cohort selection
         shell: bash
         run: |
           set -euo pipefail
           source .github/workflows/pipeline_helpers.sh
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)